apis/core/v1alpha3/flagsourceconfiguration_types.go

```diff
@@ -91,11 +91,28 @@ type FlagSourceConfigurationSpec struct {
 }
 
 type Source struct {
+	// Source is an URI of the flag sources
 	Source string `json:"source"`
+
+	// Provider type - kubernetes, http(s), grpc(s) or filepath
 	// +optional
-	Provider string `json:"provider"`
+	Provider SyncProviderType `json:"provider"`
+
+	// HttpSyncBearerToken is a bearer token. Used by http(s) sync provider only
 	// +optional
 	HttpSyncBearerToken string `json:"httpSyncBearerToken"`
+
+	// HttpSyncBearerToken is a path of a certificate to be used by grpc TLS connection(grpcs providers only)
```

Comment:
```suggestion
	// CertPath is the path of a certificate to be used by gRPC TLS connection (grpcs providers only)
```

Reply:
yes this is a copy paste error. Will fix this